Designer/Block.csv

```diff
@@ -0,0 +1,2 @@
+﻿ID,Name,AddsToBiome,GatherTool,IsFlammable,IsLiquid,MaxToughness
+-20000,Brick Block Test Parquet,None,None,False,False,0
```

Comment:
Making the maximum toughness on a block 0 feels a bit odd, and can cause some logical problems later while testing. Even if it isn't relevant now, could this be 1 instead?

Reply:
Hmm, yeah, that's a good call.
Fixed in 401be32.